frontend/vite.config.ts

```diff
@@ -13,7 +14,11 @@ const config = ({ mode }: ConfigEnv): ReturnType<typeof defineConfig> => {
     build: {
       outDir: 'build',
     },
-    plugins: [reactPlugin(), vitePWAPlugin({ registerType: 'autoUpdate' })],
+    plugins: [
+      reactPlugin(),
+      vitePWAPlugin({ registerType: 'autoUpdate' }),
+      svgr(),
```

Comment:
Please rename as the existing plugins

Reply:
Will just pulling the changes from development work? These are the changes I also picked up from cp-55 trying to get the Icon component

Reply:
Got it, but as that changes are in your PR, you need to improve it related to styles or to remove it. Check if correct naming is in latest development

Reply:
I did, what I found is that in development there are identical plugins except svgr(), I have it my branch since it's needed for the Icon component. At least, this is how initially Yevhenii Kitan set it up. I am not sure what I need to change in here.

Reply:
I guess @v0ldemar01 means it should be named like `svgrPlugin`